api/app/settings/common.py

```diff
@@ -1033,7 +1033,33 @@
 ENABLE_HUBSPOT_LEAD_TRACKING = env.bool("ENABLE_HUBSPOT_LEAD_TRACKING", False)
 HUBSPOT_IGNORE_DOMAINS = env.list("HUBSPOT_IGNORE_DOMAINS", [])
 HUBSPOT_IGNORE_DOMAINS_REGEX = env("HUBSPOT_IGNORE_DOMAINS_REGEX", "")
-
+HUBSPOT_IGNORE_ORGANISATION_DOMAINS = env.list(
+    "HUBSPOT_IGNORE_ORGANISATION_DOMAINS",
+    [
+        "gmail.com",
+        "hotmail.com",
```

Comment:
being nosey, hotmail is duplicated. This is a list I've used before if it helps https://github.com/Kikobeats/free-email-domains/blob/master/domains.json#L1

Reply:
Is this huge list being used on something at Flagsmith or for some other project? I'd just be wary of adding so many domains which are largely dormant.

Reply:
Anther project

Reply:
Anther project